packages/mongo/collection.js

```diff
@@ -674,6 +674,13 @@ Object.assign(Mongo.Collection.prototype, {
     self._collection._ensureIndex(index, options);
```

Comment:
This code should check if createIndex is available and if so it should use createIndex as well.